src/components/Settings/SettingsNewAddressbook.vue

```diff
@@ -68,14 +62,12 @@ export default {
 		/**
 		 * Set new address book name
 		 *
-		 * @param {string} addressBook The adress book
-		 * @returns {Promise}
+		 * @param {string} addressbook The address book name provided in the input
+		 *
 		 */
-		addAddressBook() {
-			let addressBook = this.$refs.addressBook.value
-			let addressBooks = this.$store.getters.getAddressbooks
-			let newAddressBooksArray = addressBooks.push(addressBook)
-			return newAddressBooksArray
+		addAddressbook() {
+			let addressbook = this.$refs.addressbook.value
+			this.$store.dispatch('appendAddressbook', { displayName: addressbook })
```

Comment:
:+1: Nice with the object that gets extended in the store!

Reply:
Thanks! :)